ProteoformSuiteInternal/TheoreticalProteoformDatabase.cs

```diff
@@ -32,6 +32,10 @@ public class TheoreticalProteoformDatabase
         //Settings
         public bool limit_triples_and_greater = true;
 
+        //Constants
+        private double ptmset_max_number_of_a_kind = 3;
```

Comment:
Consider making this `private static readonly` and changing variable to all caps, standard stuff for constants

Reply:
How about `DEFAULT_MAX_PTMS_OF_A_KIND_FOR_LOOKUP`?

Reply:
Nevermind, Microsoft uses Pascal case for these. The all caps would match the rest of ProteoformSuite, though. https://stackoverflow.com/questions/4577358/private-static-readonly-field-capitalization